ui/shared/ConfirmationDialog.qml

```diff
@@ -0,0 +1,46 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.13
+import "../imports"
+import "./"
+
+ModalPopup {
+    id: confirmationDialog
+    height: 186
+    width: 400
+    title: qsTr("Confirm your action")
+
+    property string confirmButtonLabel: "Confirm"
```

Comment:
Also, maybe we can add `readonly` to the props?
```suggestion
    property string confirmButtonLabel: qsTr("Confirm")
```

Reply:
They actually aren't readonly, because `ConfirmationDialog` shall work for different scenarios where things like `confirmButtonLabel` needs to be overwritten..